index.html

```diff
@@ -1,6 +1,24 @@
 <!DOCTYPE html>
 <html>
     <head>
+        <!-- Begin SEO tags-->
+        <meta name="description" content="Hi I'm Lemonbot and it's nice to meet you! I am a Discord bot with a lot of features like moderation, games, fun, etc." />
```

Comment:
I was looking around to see if this was something I wrote, but nothing shows up - This description is beautiful! it keeps the light-hearted tone of the website very nicely! Compliments to the chef!

Reply:
Thanks! Yeah I wrote it myself based on what I read